client/src/components/v2/CameraRecording.module.css

```diff
@@ -0,0 +1,3 @@
+.device {
+    margin-bottom: '10px'
```

Comment:
Semicolon required after this line. Not sure why eslint didn't catch it

Reply:
Done
3d2221

Not sure why eslint isn't catching it either